packages/plugin-simple-throttle/test/throttle.test.ts

```diff
@@ -3,11 +3,21 @@ import plugin from '../'
 import Client from '@bugsnag/core/client'
 
 describe('plugin: throttle', () => {
-  it('prevents more than maxEvents being sent', () => {
+  describe('addOnError', () => {
     const payloads = []
     const c = new Client({ apiKey: 'aaaa-aaaa-aaaa-aaaa' }, undefined, [plugin])
+    const mockWarn = jest.fn(message => message)
+    c._logger.warn = mockWarn
     c._setDelivery(client => ({ sendEvent: (payload) => payloads.push(payload), sendSession: () => {} }))
-    for (let i = 0; i < 100; i++) c.notify(new Error('This is fail'))
-    expect(payloads.length).toBe(10)
+
+    it('prevents more than maxEvents being sent', () => {
+      for (let i = 0; i < 100; i++) c.notify(new Error('This is fail'))
+      expect(payloads.length).toBe(10)
+      expect(mockWarn).toHaveBeenCalledTimes(90)
+    })
+
+    it('throws an appropriate error message', () => {
```

Comment:
Maybe use "logs" instead of "throws" here?

```suggestion
    it('logs an appropriate message', () => {
```